scripts/transform/index.js

```diff
@@ -6,10 +6,11 @@ var recast = require('recast');
 var transforms = {
   assertions: require('./assertions'),
   comments: require('./comments'),
+  messages: require('./messages'),
```

Comment:
I think something is missing here. You mentioned only wanting to make this transformation after tidying up the migrated files, so I'm not sure if this line should be removed or if the transform should be included in the body of the exported function.

I think I'd prefer the latter. It couldn't hurt to transform messages on the initial pass (even if we intend to re-apply the transformation later). That would also let us keep the interface of this module simple--just a single function (not a function with internal transform methods attached).

What do you think?